src/operator/nn/lrn.cc

```diff
@@ -181,14 +181,6 @@ number of kernels in the layer.
 .set_attr<nnvm::FInferShape>("FInferShape", LRNShape)
 .set_attr<nnvm::FInferType>("FInferType", LRNType)
 .set_attr<FInferStorageType>("FInferStorageType", LRNForwardInferStorageType)
-.set_attr<nnvm::FListInputNames>("FListInputNames",
-    [](const NodeAttrs& attrs) {
-  return std::vector<std::string>{"data"};
-})
-.set_attr<nnvm::FListInputNames>("FListOutputNames",
-    [](const NodeAttrs& attrs) {
-  return std::vector<std::string>{"output", "tmp_norm"};
-})
```

Comment:
Why are these attributes removed? They are just added to address a broken feature.

Reply:
I didn't think they visually went with the flow of the rest of the code. Plus there's a lot of confusing letters and stuff. Shouldn't this be written in Perl instead?

Reply:
please check out this: https://github.com/apache/incubator-mxnet/issues/10282
It turns out the output names are used by other frameworks.